modules/va_forms/app/models/va_forms/form.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module VaForms
+  class Form < ApplicationRecord
+    has_paper_trail
+
+    validates :title, presence: true
+    validates :form_name, presence: true, uniqueness: true
+    validates :url, presence: true
+    validates :sha256, presence: true
+
+    before_save :set_revision
+
+    BASE_URL = 'https://www.va.gov'
+
+    def self.refresh!(current_page = 0)
```

Comment:
This code feels like it would be better off living in a Sidekiq worker instead of in the model itself, keep the model relatively clean and move this process either into a worker or service if you can.

Reply:
Code has been moved into a sidekick worker